Platform/AlderlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.h

```diff
@@ -107,4 +107,15 @@ RtcInit (
   VOID
   );
 
+/**
+  This function does the basic init work for Max6950.
+
+  @param[in]  SpiBaseAddress - Base Address of SPI Controller
+**/
+EFI_STATUS
+EFIAPI
+MaxLedInit (
```

Comment:
This is a module internal function, EFIAPI is not required.

Reply:
done